Sming/SmingCore/Network/TcpClient.h

```diff
@@ -54,6 +54,13 @@ class TcpClient : public TcpConnection
 	__forceinline bool isProcessing()  { return state == eTCS_Connected || state == eTCS_Connecting; }
 	__forceinline TcpClientState getConnectionState() { return state; }
 
+	inline void setReceiveCallback( TcpClientDataDelegate on_receive ) {
+		receive = on_receive;
+	}
+	inline void setCompleteCallback( TcpClientCompleteDelegate on_complete ) {
```

Comment:
> I missed that somehow becaused it is not mentioned in the api documentation ...

You have a perfect chance to fix that by adding proper documentation comments :)

> Or should the new method go to TcpClient.cpp as setCompleteDelegate()?

I would prefer that.